fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -369,16 +368,13 @@ private static String getRelativePath(final FedoraResource child, final FedoraRe
      * @return
      */
     private String doBackwardPathOnly(final FedoraResource resource) {
-        String path = reverse.convert(getPath(resource));
+        final String path = reverse.convert(getPath(resource));
         if (path != null) {
 
             if (resource instanceof NonRdfSourceDescription) {
-                path = path + "/" + FCR_METADATA;
+                return path + "/" + FCR_METADATA;
             }
 
-            if (path.endsWith(JCR_CONTENT)) {
```

Comment:
`FedoraResource::getPath` will never end in `JCR_CONTENT`. See this: https://git.io/vrhaU (it was a somewhat recent change).